app/addons/documents/index-results/components/results/TableRow.js

```diff
@@ -93,7 +93,7 @@ export default class TableRow extends React.Component {
     }
 
     return (
-      <td className="tableview-el-last" onClick={this.onClick}>
+      <td className="tableview-el-last" onClick={this.onClick.bind(this)}>
```

Comment:
Shouldn't you bind the onClick in the constructor to be more optimal? Binding in render() functions will be called  for each render(which is not optimal).

Reply:
@popojargo good idea. I'm not going to make the change for this in this PR since its not related. Could you do a PR for this. I will update all the other binds in the rev-browser section

Reply:
@garrensmith Sure, I'll work on it after the setup pr :)